src/main/java/com/imsweb/mph/mpgroups/Mp2023OtherSitesGroup.java

```diff
@@ -22,6 +22,7 @@
 import com.imsweb.mph.mprules.MpRuleNoCriteriaSatisfied;
 import com.imsweb.mph.mprules.MpRulePrimarySite;
 
+@SuppressWarnings("java:S3776")
```

Comment:
You suppressed a couple of these, and I am OK with that. But can you please put a comment after it explaining what it was (and a reason for the suppression if it's not obvious). It's a pain to have to look online for what S3776...